docs/usage/self-hosted-configuration.md

```diff
@@ -131,6 +131,7 @@ By default, Renovate uses a child process to run such tools, so they must be:
 
 - installed before running Renovate
 - available in the path
+- managed by Hermit
```

Comment:
I don't think this is the right place. The text above it is describing binarySource=global